### DISCUSSION TITLE
Is meta-tegra registered  as Yocto Project® Compatible ?

### DISCUSSION THREAD

**Discussion by fischer** [OE4T/meta-tegra#446] [General]
Apologies in advance for wasting time on a non-technical topic.

Has anyone filled out the Yocto compatible form for meta-tegra at https://www.yoctoproject.org/ecosystem/branding/compatible-registration ?

I don't feel comfortable representing meta-tegra, or I would fill it out.  As far as I can tell, using the word *Yocto* in the README.md file requires compatibility registration.  From https://www.yoctoproject.org/logos-and-guidelines :

> Any use of {Y,y}-o-{C,c}-{T,t}-o Project or any subset of those letters or sounds in that order, alone or in combination with other letters, words or marks that would tend to cause someone to make a reasonable connection with the Yocto Project has to be in reference to code that is in conformance with the Yocto Project Compliance Program.

**Comment by koansoftware** [comment]
Hi Todd,
why you don't feel comfortable representing meta-tegra?
You (or your company) are the creator and mantainer of the project so probably you would be the main responsible for it.
The registration process is very easy and pain free.
Feel free to ask me anything you consider not clear enough.
Best,

_Marco Cavallini
{ Yocto Project Ambassador }
{ KOAN  C.E.O. }
Bergamo - Italia
embedded software engineering
https://KoanSoftware.com_

**Comment by madisongh** [comment]
> Has anyone filled out the Yocto compatible form for meta-tegra at https://www.yoctoproject.org/ecosystem/branding/compatible-registration ?

Not yet... branding hasn't really been on my radar so far, but I'll try submitting an entry to see what they say.

> You (or your company) are the creator and mantainer of the project so probably you would be the main responsible for it.

Ahem, but no, that is incorrect.

**Comment by fischer** [comment]
Matt,

The form looks to take around 5 minutes to fill out.  They at least made it easy.

Thanks for looking into it.

Todd

**Comment by madisongh** [comment]
OK, I've submitted both a 'Participant' form for the project, and a 'Compatible' form for the layer itself.  I'll post here if/when I get a response.

**Comment by koansoftware** [comment]
@madisongh very good!
I'm sure you'll get a reply from the Yocto Project team soon.

/marco

**Comment by madisongh** [comment]
The project is now a registered Yocto Project participant.  (All we need now is a logo to replace the text on [the Participants page](https://www.yoctoproject.org/ecosystem/participants/).)

The dunfell and gatesgarth branches will be reviewed by the TSC for the 'Compatible' mark, and Nico said he'd see if we can get the same for the dunfell-l4t-r32.4.3 branch, but since it's not just named 'dunfell', it may not be eligible.  We'll have to bear that in mind going forward, and each future release branch will also need to be submitted for the 'compatible' mark.  It should be easier for the subsequent approvals, though.

**Reply by koansoftware** [reply]
Great news @madisongh welcome on board! 

Marco Cavallini
{ Yocto Project Ambassador }
